### PR TITLE
Update projects.html

### PR DESCRIPTION
**Pull request by mrwangkai**

### CODE REVIEW

File: projects.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	<body>
 		<div class="nav-top">
 			
-			<a href="index.html" id="namesakeHeader">Kai Wang</a>
```
Comment (ansario) [inline projects.html:24]: This might be a typo.

**Review by ansario — CHANGES_REQUESTED** [review]

File: projects.html (hunk 1 above)
Comment (mrwangkai) [reply projects.html:24]: Got it, will fix.

File: projects.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 		<meta http-equiv="Content-Type" content="<?php bloginfo('html_type'); ?>; charset=<?php bloginfo('charset'); ?>" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		
-		<title>JLO / About</title>
+		<title>JLO / About_</title>
```
Comment (ansario) [inline projects.html:7]: We need to fix this.

**Review by ansario — CHANGES_REQUESTED** [review]

File: projects.html (hunk 2 above)
Comment (mrwangkai) [reply projects.html]: fixed!

**Review by ansario — APPROVED** [review]

**Pull request closed without merging**